calendar-tool/tool/src/webapp/vm/calendar/chef_calendar_icalexport.vm

```diff
@@ -25,7 +25,7 @@
 	</h3>
 	
 	<p class="messageInformation">
-		$tlang.getString("ical.info")<br/>
+		$icalInfoStr $tlang.getString("ical.info")<br/>
```

Comment:
This doesn't look right, `ical.info` needs substituting when output, should it be `ical.info.public`?